asistente_ladm_col/gui/supplies/dlg_missing_snc_supplies.py

```diff
@@ -0,0 +1,309 @@
+                'predio': self.alphanumeric_file_paths['PREDIO'],
+                'rconstruccion': self.gdb_layer_paths['R_CONSTRUCCION'],
+                'rterreno': self.gdb_layer_paths['R_TERRENO'],
+                'runidad': self.gdb_layer_paths['R_UNIDAD'],
+                'uconstruccion': self.gdb_layer_paths['U_CONSTRUCCION'],
+                'uterreno':self.gdb_layer_paths['U_TERRENO'],
+                'uunidad': self.gdb_layer_paths['U_UNIDAD'],
+                'vereda': self.gdb_layer_paths['R_VEREDA'],
+                'qgis:refactorfields_1:COMISIONES_TERRENO':'TEMPORARY_OUTPUT',
+                'qgis:refactorfields_2:OMISIONES_TERRENO':'TEMPORARY_OUTPUT',
+                'qgis:refactorfields_3:COMISIONES_MEJORAS':'TEMPORARY_OUTPUT',
+                'qgis:refactorfields_4:OMISIONES_MEJORAS':'TEMPORARY_OUTPUT',
+                'qgis:refactorfields_5:COMISIONES_UNID_PH':'TEMPORARY_OUTPUT',
+                'qgis:refactorfields_6:OMISIONES_UNID_PH':'TEMPORARY_OUTPUT',
+                'qgis:refactorfields_7:COMISIONES_MZ':'TEMPORARY_OUTPUT',
+                'qgis:refactorfields_8:OMISIONES_MZ':'TEMPORARY_OUTPUT',
+                'qgis:refactorfields_9:COMISIONES_VR':'TEMPORARY_OUTPUT',
+                'qgis:refactorfields_10:OMISIONES_VR':'TEMPORARY_OUTPUT'},
+                                                           feedback=self.custom_feedback)
+        except QgsProcessingException as e:
+            if self.custom_feedback.isCanceled():
+                # The algorithm can throw errors even if canceled, so catch them and if we canceled, silent them.
+                return False, 'Missing cobol supplies algorithm canceled!'
+            else:
+                msg = "QgsProcessingException (Missing cobol supplies): {}".format(str(e))
+                self.logger.critical(__name__, msg)
+                self.show_message(msg, Qgis.Critical)
+
+        self.logger.info(__name__, "Missing Cobol Supplies model finished.")
+
+    def package_results(self, output):  
+        for name in output.keys():
+            output[name].setName(name.split(':')[2])
+
+        try:
+            output_geopackage = processing.run("native:package", {'LAYERS': [
+                        output['qgis:refactorfields_1:COMISIONES_TERRENO'],
+                        output['qgis:refactorfields_2:OMISIONES_TERRENO'],
+                        output['qgis:refactorfields_3:COMISIONES_MEJORAS'],
+                        output['qgis:refactorfields_4:OMISIONES_MEJORAS'],
+                        output['qgis:refactorfields_5:COMISIONES_UNID_PH'],
+                        output['qgis:refactorfields_6:OMISIONES_UNID_PH'],
+                        output['qgis:refactorfields_7:COMISIONES_MZ'],
+                        output['qgis:refactorfields_8:OMISIONES_MZ'],
+                        output['qgis:refactorfields_9:COMISIONES_VR'],
+                        output['qgis:refactorfields_10:OMISIONES_VR']],
```

Comment:
What if you simply use `{'LAYERS': output.keys()}`  ?

Reply:
OMG, Of course, this is better, I changed it with this:

    list(output.values())

Thanks